packages/cli/src/cmds/validator/handler.ts

```diff
@@ -165,7 +166,9 @@ export async function validatorHandler(args: IValidatorCliArgs & GlobalArgs): Pr
       doppelgangerProtectionEnabled,
       afterBlockDelaySlotFraction: args.afterBlockDelaySlotFraction,
       scAfterBlockDelaySlotFraction: args.scAfterBlockDelaySlotFraction,
+      disableAttestationGrouping: args.disableAttestationGrouping,
```

Comment:
I introduced this option with the intention to replace `distributed`. Having both options do not make sense. Pick one or the other. I don't have strong feeling towards one or the other

Reply:
My reasoning behind this is the following

- `distributed` is specific to running the VC in a distributed validator cluster, this flag will enable further features later, specifically talking about [aggregate committee selections](https://docs.google.com/document/d/1q9jOTPcYQa-3L8luRvQJ-M0eegtba4Nmon3dpO79TMk/mobilebasic)
- `disableAttestationGrouping` is to disable the grouping optimization individually. This gives us more flexibility if there are other similar compatibility issues in the future.

If we just keep `disableAttestationGrouping` we would need another flag to enable aggregate committee selections and be fully dvt compliant. 

Therefore, we either keep both or just `distributed`.

Reply:
@dapplion what do you think about using `distributed` as cli option but still keeping `disableAttestationGrouping` as option in attestation service and then just setting that to `true` if `distributed=true`?